webrtc-stats.html

```diff
@@ -1066,6 +1078,27 @@
                   section 6.4.1.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>lastPacketSentTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the timestamp at which the last packet was sent for this media stream track.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>averageRTCPInterval</code></dfn> of type <span class=
```

Comment:
Isn't when the RTCP report were received, [timestamp](https://w3c.github.io/webrtc-pc/#dom-rtcstats-timestamp)? The [guideline](https://w3c.github.io/webrtc-stats/#guidelines-for-design-of-stats-objects) says "no averages".